sound/soc/sof/ipc4-pcm.c

```diff
@@ -465,10 +465,7 @@ static int sof_ipc4_pcm_dai_link_fixup(struct snd_soc_pcm_runtime *rtd,
 	struct snd_interval *rate = hw_param_interval(params, SNDRV_PCM_HW_PARAM_RATE);
 	struct snd_mask *fmt = hw_param_mask(params, SNDRV_PCM_HW_PARAM_FORMAT);
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(component);
-	struct snd_soc_dai *cpu_dai = asoc_rtd_to_cpu(rtd, 0);
```

Comment:
commit message:

DO NOT USE "IT", even worse DO NOT REPEAT "IT" three times in the same sentence with 

````
It introduces a problem that it uses more
audio bus bandwidth when 16 bits stream is required by application in
some cases like feedback processing and it will be failed to support
such case for inefficient bandwidth on soundwire platform
````

"IT" is the worst possible subject in technical write-ups. No one has a clue  what 'it' refers to.

Reply:
sure, thanks